pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[project]
+name = "comfyui_isaacnodes"
+description = "This extension offers automatic drums extraction from audio files, as well as a few helper nodes to support my audio synchronization AnimateDiff workflows."
+version = "1.0.0"
+license = "LICENSE"
```

Comment:
Should this be `COPYING`? Does the license get imported?

Reply:
so the file path is relative to the pyproject.toml file so if it's in the root directory it'll be the path to wherever your toml file, so yes it be `{file = COPYING}`